textdb/textdb-api/src/main/java/edu/uci/ics/textdb/api/common/FieldType.java

```diff
@@ -8,5 +8,8 @@
 	DOUBLE, 
 	DATE,
 	// A field that is indexed and tokenized,without term vectors
-	TEXT;
+	TEXT,
+
+	//A field that is the list of strings
+	STRING_LIST;

```

Comment:
We need to use the generic list field here. Once our pull request #77 is merged into master, you can make these changes